schema/scan.go

```diff
@@ -214,6 +214,36 @@ func scanUint64(dest reflect.Value, src interface{}) error {
 	}
 }
 
+func scanFloat32(dest reflect.Value, src interface{}) error {
+	switch src := src.(type) {
+	case nil:
+		dest.SetFloat(0)
+		return nil
+	case float32:
+		dest.SetFloat(float64(src))
+		return nil
+	case float64:
+		dest.SetFloat(src)
+		return nil
+	case []byte:
+		f, err := strconv.ParseFloat(internal.String(src), 64)
```

Comment:
Probably should be 64 -> 32

Reply:
I made a revision; perhaps we don’t need to differentiate here anymore.